types/tar/index.d.ts

```diff
@@ -307,7 +307,7 @@ export interface PackOptions {
      * Set to a Date object to force a specific mtime for everything added to
      * the archive. Overridden by noMtime.
      */
-    mtime?: number;
+    mtime?: Date;
```

Comment:
Hi @SomaticIT, @connor4312! Do we need to increment minor version introducing this change?

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/945cb7690e8fb6989b3ccb6b4a6fdc9315f05d2c/types/tar/package.json#L4

```diff
-    "version": "6.1.9999",
+    "version": "6.2.9999", 
```